wowup-electron/src/app/services/wowup/wowup.service.ts

```diff
@@ -245,8 +266,9 @@ export class WowUpService {
 
     // only notify things when the version changes
     if (
-      updateCheckResult.updateInfo.version !==
-      this._electronService.getVersionNumber()
+      updateCheckResult && 
+      updateCheckResult.updateInfo && 
```

Comment:
Because I was returning an empty object that wouldn't work. I've changed to returning a null and then just check for base object and then safe navigation to the version property.